examples/using-gatsby-a11y-helpers/README.md

```diff
@@ -0,0 +1,102 @@
+    <Header />
+    <SkipNavContent /> //the link will go to here, so put it right before main content
+    <main>{children}</main>
+    <Footer />
+  </>
+)
+export default Layout
+```
+
+### Focus your link on page navigation
+
+Hooking into Gatsby's `onRouteUpdate` API method will allow you to focus automatically on a skip link on page change, putting a user in a more appropriate spot to take action. More information about this method can be found the [browser API docs](https://www.gatsbyjs.org/docs/browser-apis/#onRouteUpdate).
+
+```javascript:title=gatsby-browser.js
+// gatsby-browser.js
+
+export const onRouteUpdate = ({ location, prevLocation }) => {
+  if (prevLocation !== null) {
+    const skipLink = document.querySelector("[data-reach-skip-link]") //this is the query selector that comes with the <SkipNavLink> component
+    if (skipLink) {
+      skipLink.focus()
+    }
+  }
+}
+```
+
+## 🚀 Using this example
+
+### 🧐 What's inside?
+
+A quick look at the relevant files and directories you'll see in this example:
+
+```text
+    .
+    ├── cypress/
+    │   ├── integration/
+    │   │   ├── skip-nav.test.js
+    ├── src/
+    │   ├── components/
+    │   │   ├── header.css
+    │   │   ├── header.js
+    │   │   ├── layout.js
+    │   │   └── seo.js
+    │   └── pages/
+    ├── gatsby-browser.js
+```
+
+1.  **`/src`**: This directory will contain all of the code related to what you will see on the front-end of your site. It has pages and components to be used in those pages.
+1.  **`/components`**: This directory will contain all of the code related to what you will see on the front-end of your site. It has pages and components to be used in those pages.
+
+    1.  **`/header.css`**: This ensures that our navigation links have focus rings, which are important visual cues when navigating by keyboard.
+    1.  **`/header.js`**: This has the navigation links for your pages.
+    1.  **`/layout.js`**: This component is used in all of your pages to make sure they all use the same navigation. This is also where you will implement your skip link.
+    1.  **`/seo.js`**: This information is necessary to give the site a 100% in [Lighthouse](https://developers.google.com/web/tools/lighthouse) for accessibility since screen readers need a title and language.
+
+1.  **`/pages`**: This directory contains pages that will be automatically built and served by Gatsby. This example includes three pages to demonstrate navigation between them and how your skip nav link behaves. All of these pages use the `Layout` component.
+1.  **`gatsby-browser.js`**: This file is where you tell the Gatsby to focus the skip navigation when users navigate to a new page.
+1.  **`/cypress`**: This directory is where tests and [Cypress](https://www.cypress.io/) configuration live. You're going to focus on the test. If you want to learn more about using Cypress, check out the [example](https://github.com/gatsbyjs/gatsby/tree/master/examples/using-cypress).
```

Comment:
"You're going to focus on the test". This is a bit confusing. Do you mean that the test makes use of navigation focus?